### PR TITLE
CVSB-1242 - Change technical record model

### PR DESCRIPTION
**Pull request by acozianu**

### CODE REVIEW

File: src/models/vehicle/tech-record.model.ts [hunk 1]
```diff
@@ -87,3 +88,13 @@ export interface TyresModel {
   speedCategorySymbol: string;
   tyreCode: number;
 }
+
+export interface BodyType {
```
Comment (vladddragomir) [inline src/models/vehicle/tech-record.model.ts]: add Model to interface's name ex: BodyTypeModel

File: src/models/vehicle/tech-record.model.ts [hunk 2]
```diff
@@ -87,3 +88,13 @@ export interface TyresModel {
...
+  code: string;
+  description: string;
+}
+
+export interface vehicleClass {
```
Comment (vladddragomir) [inline src/models/vehicle/tech-record.model.ts]: same here

File: src/models/vehicle/tech-record.model.ts (hunk 1 above)
Comment (acozianu) [reply src/models/vehicle/tech-record.model.ts]: Fixed

File: src/models/vehicle/tech-record.model.ts (hunk 2 above)
Comment (acozianu) [reply src/models/vehicle/tech-record.model.ts]: Fixed

**Pull request merged**